packages/use-query-string/src/useQueryString.ts

```diff
@@ -11,6 +11,11 @@ export const withQueryString = async (
   mapper: (validatedData: any) => any,
   props: Props
 ) => {
+  if (!props.request.query) {
+    throw new BadRequestError('Missing query string', {
+      key: 'err.query-string.failed'
```

Comment:
The `key` property is meant to be unique so it can always be tracked to a specific line/throw. Maybe `err.query-string.empty` instead?